.gitignore

```diff
@@ -0,0 +1 @@
+bootstrap*.tar.gz
```

Comment:
This will upload any `bootstrap*.tar.gz` to Docker context at `docker build` time, even if you decide to `ADD` only one of them in the Dockerfile. I don't want that. I wan't to upload and `ADD` `bootstrap.tar.gz` only, ever, to keep `docker build` simple and fast.